00-HTML-CSS-basics/HTML/Excersise1/media.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="autor" content="Demian-Moschin">
+    <meta name="description" content="Media content for studie purpose only">
+    <meta name="viewport" content="width=device-width, user-scalable=no,initial-scale=1" >
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" >
+
+    <title>Audio and video</title>
+
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+      <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+      <!--[if lt IE 9]>
+        <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+        <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+      <![endif]-->
+      
+  </head>
+
+  <body>
+    
+
+    <section>
+      <header>
+        <h1>Videos</h1>
+      </header>
+      <video controls autoplay loop muted>
+        <source src="media/jquery.webm" type="video/webm" />
```

Comment:
You'll need to add some other audio & video formats, to keep browser compatibility. The link provided into the repo it's enought information I think.